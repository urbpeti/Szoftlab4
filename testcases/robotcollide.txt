Robot R1 35 -3
Robot R2 31 1
Step
List_Robots