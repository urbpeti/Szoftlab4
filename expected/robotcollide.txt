R1, Position: 32.0, Velocity: -1.0, Not Oily, Alive
R2, Position: 32.0, Velocity: 1.0, Not Oily, Dead
